modules/core_api/fsw/inc/cfe_version.h

```diff
@@ -28,21 +28,20 @@
 #define CFE_VERSION_H
 
 /* Development Build Macro Definitions */
-#define CFE_BUILD_NUMBER   80           /**< @brief Development: Number of development commits since baseline */
+#define CFE_BUILD_NUMBER   80 /**< @brief Development: Number of development git commits since CFE_BUILD_BASELINE */
 #define CFE_BUILD_BASELINE "v7.0.0-rc4" /**< @brief Development: Reference git tag for build number */
 
-/* Version Macro Definitions updated for official releases only */
-#define CFE_MAJOR_VERSION 6 /**< @brief Major release version (Former for Revision == 99) */
-#define CFE_MINOR_VERSION 7 /**< @brief Minor release version (Former for Revision == 99) */
-#define CFE_REVISION \
-    99 /*!< @brief * Set to 0 on OFFICIAL releases, and set to 99 on development versions. Revision number.      */
+/* See cfs_versions.dox for definitions */
```

Comment:
Is there a doxygen "link command" of sorts to turn this into a hyperlink in the generated documentation?

Reply:
yes, although you need to link to the page name... see https://github.com/nasa/CF/blob/d0b3e25566b7cb8221d8a3f90017fe52ee0f43d4/fsw/src/cf_version.h#L31